core/src/main/resources/jenkins/model/Jenkins/legend.jelly

```diff
@@ -25,11 +25,11 @@ THE SOFTWARE.
 
 <!-- show the icon legend -->
 <?jelly escape-by-default='true'?>
-<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:s="/lib/form">
-  <l:layout title="${%Icon legend}">
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson"
+         xmlns:s="/lib/form" xmlns:f="/lib/form">
+  <l:layout title="${%Icon legend}" type="one-column">
     <l:app-bar title="${title}" />
```

Comment:
LGTM! Would you mind moving the app-bar inside of main-panel, like [this](https://weekly.ci.jenkins.io/design-library/Layouts/).

Reply:
For sure